sharding/common.go

```diff
@@ -4,6 +4,32 @@ import (
 	"github.com/ElrondNetwork/elrond-go/core"
 )
 
+func cloneValidatorsMap(validatorsMap map[uint32][]Validator) (map[uint32][]Validator, error) {
+	var err error
+	resultMap := make(map[uint32][]Validator)
+	for shard, vList := range validatorsMap {
+		resultMap[shard], err = cloneValidatorsList(vList)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return resultMap, nil
+}
+
+func cloneValidatorsList(validatorsList []Validator) ([]Validator, error) {
+	var err error
+	var clone interface{}
+	resultList := make([]Validator, len(validatorsList))
+	for i, v := range validatorsList {
+		clone, err = v.Clone()
```

Comment:
Why not: clone, err := v.Clone() and remove lines 20 and 21

Reply:
done